core/services/fluxmonitor/flux_monitor.go

```diff
@@ -666,14 +666,29 @@ func (p *PollingDeviationChecker) checkEligibilityAndAggregatorFunding(roundStat
 		return ErrNotEligible
 	} else if roundState.AvailableFunds.Cmp(roundState.PaymentAmount) < 0 {
 		return ErrUnderfunded
-	} else if roundState.PaymentAmount.Cmp(p.store.Config.MinimumContractPayment().ToInt()) < 0 {
+	} else if p.SufficientPayment(roundState.PaymentAmount) {
```

Comment:
Looks like the logic here is reversed -- it should be `!p.SufficientPayment`, right?

Reply:
Great catch!